src/database/migrations/20191016081528-create-rating.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Ratings', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      user: {
```

Comment:
since `user` this is a foreign key in this table it should be indicated in the migration and model